src/Generator/TypeGenerator.php

```diff
@@ -24,18 +24,22 @@
     private $type;
 
     /**
+     * @var bool
+     */
+    private $nullable;
+
+    /**
      * @var string[]
      *
      * @link http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration
      */
-    private static $internalPhpTypes = ['int', 'float', 'string', 'bool', 'array', 'callable'];
+    private static $internalPhpTypes = ['void', 'int', 'float', 'string', 'bool', 'array', 'callable'];
```

Comment:
Alright, to be added (as separate test asset, since `iterable` can be a class/interface in 5.6 and 7.0